service/lib/agama/storage/config_conversions/to_json_conversions/search.rb

```diff
@@ -33,13 +33,22 @@ def self.config_type
             Configs::Search
           end
 
+          # @see Base#convert
+          def convert
+            converted = super
+            return SEARCH_ANYTHING_STRING if converted.is_a?(Hash) && anything?(converted)
```

Comment:
I am not sure yet if we should generate shortcuts. I think this is the only case in which we do it. Let's keep it for now.

Reply:
I would say that adding the device name is kind of expected for a solved config. I mean, independently of the initial  conditions, I would expect the solved config should be a search by name. For example:

```json
{ 
  search: { 
    condition: { size: { bigger: "10 GiB" } }
  }
}
```

Resolves to:

```json
{ 
  search: "/dev/vdc" 
}
```

But of course, we can reconsider it.